### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Ice94**
Add simple views using JavaFx.

### CODE REVIEW

File: src/main/java/com/fourmusketeers/client/View/WelcomeWindow.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        gridPane.setAlignment(Pos.CENTER);
+        gridPane.setVgap(20);
+
+        gridPane.add(createServerButton(), 0, 0);
+        gridPane.add(createJoinButton(), 0, 1);
+        scene = new Scene(gridPane, Main.SCREEN_WIDTH, Main.SCREEN_HEIGHT);
+    }
+
+    private Button createJoinButton() {
+        final String GAME_ICON_URL = "http://icons.iconarchive.com/icons/treetog/i/128/Games-icon.png";
```
Comment (patkkka) [inline src/main/java/com/fourmusketeers/client/View/WelcomeWindow.java]: Nice icons :)

File: src/main/java/com/fourmusketeers/client/View/GameWindowView.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            }
+
+        }
+    }
+
+    private void addMyBoardToGrid(GridPane myBoardPane) {
+        for (int i = 0; i < Main.BOARD_SIZE; i++) {
+            for (int j = 0; j < Main.BOARD_SIZE; j++) {
+                //TODO get board from somewhere
+                myBoardPane.add(new Button(), j, i);
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/View/GameWindowView.java]: Which is vertically and which is horyzontally?

File: src/main/java/com/fourmusketeers/client/View/WelcomeWindow.java (hunk 1 above)
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/View/WelcomeWindow.java]: Should be static resource.

File: src/main/java/com/fourmusketeers/client/View/WelcomeWindow.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        Button connectToServerButton = new Button("Connect to game", new ImageView(GAME_ICON_URL));
+        connectToServerButton.setMaxWidth(Double.MAX_VALUE);
+        connectToServerButton.setOnAction(event -> new PropertiesWindowView(primaryStage, false, this).display());
+        return connectToServerButton;
+    }
+
+    private Button createServerButton() {
+        final String SERVER_ICON_URL = "http://icons.iconarchive.com/icons/icons-land/vista-hardware-devices/128/Home-Server-icon.png";
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/View/WelcomeWindow.java]: Should be static resource.

File: src/main/java/com/fourmusketeers/client/View/PropertiesWindowView.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        Label serverLabel = new Label("IP: ");
+        serverIPField = new TextField("localhost");
+        Label portLabel = new Label("Port: ");
+        TextField portField = new TextField("4444");
+        HBox box = new HBox(20);
+        box.getChildren().add(createNextButton());
+        box.setAlignment(Pos.BOTTOM_RIGHT);
+        HBox prevBox = new HBox(20);
+        prevBox.getChildren().add(createPreviousButton());
+        prevBox.setAlignment(Pos.BOTTOM_LEFT);
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/View/PropertiesWindowView.java]: A little unreadable code, think about spaces or dividing to few method.

File: src/main/java/com/fourmusketeers/client/View/ShipPlacementWindowView.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import javafx.scene.control.Button;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.stage.Stage;
+
+/**
+ * @author paul
+ * @since 25.07.17
+ */
+class ShipPlacementWindowView extends WindowView {
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/View/ShipPlacementWindowView.java]: Think about bigger buttons ;)

**Pull request merged**